app/src/main/res/layout/pref_slider.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (C) 2018 The Android Open Source Project
```

Comment:
Is this code simply copied from somewhere?

The comment above "better than SeekBar" sounds different?

(Is the license necessary?)

Reply:
Yes. As noted on line 18.

The source is https://android.googlesource.com/platform/frameworks/support/+/refs/heads/androidx-main/preference/preference/res/layout/preference_widget_seekbar_material.xml, from AOSP. This just has minimal changes to swap in Slider instead of Seekbar, and provide the optional increment/decrement buttons.